redi-connect-front/src/components/organisms/RediHero.tsx

```diff
@@ -18,20 +13,22 @@ const RediHero = () => {
   return (
     <Section className="default-background">
       <Container>
-        <Columns vCentered>
-          <Columns.Column size={5}>
+        <Columns>
+          <Columns.Column size={5} className="redi-hero__content">
```

Comment:
This says that `redi-hero__content` is a part of `redi-hero`. I don't se the `redi-hero` class anywhere in the the component. 

https://en.bem.info/methodology/

Reply:
I would not recommend separating even class dependencies across multiple components.

Reply:
removed